packages/react-app/src/components/ChallengeExpandedCard.jsx

```diff
@@ -151,7 +196,7 @@ const ChallengeExpandedCard = ({
         pl={10}
         borderLeft="solid 5px"
         borderColor={borderColor}
-        borderBottom={isLast ? 0 : "2px"}
+        borderBottom={isLast || challengeIndex === 3 ? 0 : "2px"}
```

Comment:
Sorry for this 😬 

Another alternative might be giving a small negative margin to BgBanner Card so that it overlaps with the border, please let me know which one we should go for or any other solution 🙌

Reply:
:eyes: haha

Added this comment 4244efa224d0267d0286807ba707fa1312b679e9